.kokoro/.mocharc.yml

```diff
@@ -0,0 +1,3 @@
+reporter: xunit
```

Comment:
will we still get human readable output in the kokoro logs as well? If output is set to a file.

Reply:
Yes, if the `sponge_log.xml` exporting is set up (as it is in this PR for `appengine` and `functions`).

⚠️ **N.B:** we should do one of the following, to avoid breaking _other_ tests:
1. add the XML export config to the top-level Kokoro config
2. only use the `.mocharc.yml` file for `functions` and `appengine` tests.

(The first option is a bit easier to implement, but it would create more UI changes in the short term.)

Reply:
Choosing option 1 - asked @fhinkel for LGTM below. 🙂